fluentd/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM fluent/fluentd:v1.2.1-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+EXPOSE 24224/udp
+
+CMD exec fluentd -c /fluentd/etc/${FLUENTD_CONF} -p /fluentd/plugins $FLUENTD_OPT -v
```

Comment:
This line seems to support the same environment variables like `fluent/fluentd:v1.2.1-onbuild` itself. Why do we need this extra Dockerfile?